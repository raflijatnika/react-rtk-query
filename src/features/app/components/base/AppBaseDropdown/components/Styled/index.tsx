// Styled Components
import styled from 'styled-components'

// Antd
import { Dropdown } from 'antd'

export const StyledDropdown = styled(Dropdown)``
