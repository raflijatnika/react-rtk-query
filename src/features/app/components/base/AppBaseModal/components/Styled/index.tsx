// Styled Components
import styled from 'styled-components'

// Antd
import { Modal } from 'antd'

export const StyledModal = styled(Modal)``
