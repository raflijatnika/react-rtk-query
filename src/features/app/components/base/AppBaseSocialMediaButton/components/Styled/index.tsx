// Styled Components
import styled from 'styled-components'

const StyledWrapper = styled.div`
  display: flex;
  align-items: center;
  justify-content: space-between;
  margin-top: 20px;
  gap: 20px;

  @media (max-width: 1024px) {
    justify-content: space-evenly;
  }
`

export { StyledWrapper }
