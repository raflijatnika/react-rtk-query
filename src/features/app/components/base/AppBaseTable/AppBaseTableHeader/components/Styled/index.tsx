// Styled Components
import styled from 'styled-components'

export const StyledSelectOptionWrapper = styled.div`
  width: 20%;

  @media (max-width: 1024px) {
    width: 100%;
  }
`
