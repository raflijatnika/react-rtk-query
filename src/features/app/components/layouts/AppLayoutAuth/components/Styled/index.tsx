// Styled Components
import styled from 'styled-components'

const StyledWrapper = styled.div`
  display: flex;
  align-items: center;
  height: 100vh;
  width: 100vw;
`

const StyledContainer = styled.div`
  display: flex;
  align-items: center;
  justify-content: center;
  width: 100%;
  height: 100%;
`

const StyledFormContainer = styled.div`
  width: 30vw;
  height: 100%;
  background-color: #fff;
  display: flex;
  justify-content: center;
  flex-direction: column;
  padding: 0px 68px;

  @media (max-width: 1024px) {
    width: 100vw;
    padding: 0px 30px;
  }
`

const StyledBanner = styled.div`
  background-color: #f1f4fa;
  width: 70vw;
  height: 100%;
  display: flex;
  align-items: center;
  justify-content: center;

  @media (max-width: 1024px) {
    display: none;
  }
`

export { StyledWrapper, StyledContainer, StyledFormContainer, StyledBanner }
