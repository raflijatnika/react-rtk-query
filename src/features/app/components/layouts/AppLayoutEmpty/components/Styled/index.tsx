// Styled Components
import styled from 'styled-components'

// Constant
import { APP_COLOR_LIGHT } from '@/features/app/constant/app-style.constant'

const StyledWrapper = styled.div`
  width: 100vw;
  height: 100vh;
  background-color: ${APP_COLOR_LIGHT.BACKGROUND};
`

export { StyledWrapper }
