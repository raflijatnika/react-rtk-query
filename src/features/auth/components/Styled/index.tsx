// Styled Components
import styled from 'styled-components'

const StyledCredentialsContainer = styled.div`
  display: flex;
  align-items: center;
  justify-content: space-between;
  margin-top: 30px;
`

export { StyledCredentialsContainer }
