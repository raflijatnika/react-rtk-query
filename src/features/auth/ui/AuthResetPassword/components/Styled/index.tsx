// Styled Components
import styled from 'styled-components'

// Constant
import { APP_COLOR } from '@/features/app/constant/app-style.constant'

const StyledWrapper = styled.div`
  height: 100%;
  display: flex;
  align-items: center;
  justify-content: center;
  flex-direction: column;
`

const StyledCard = styled.div`
  background-color: ${APP_COLOR.WHITE};
  width: 658px;
  height: 658px;
  display: inherit;
  justify-content: inherit;
  flex-direction: inherit;
  border-radius: 10px;
  padding: 0px 80px;
`

export { StyledWrapper, StyledCard }
