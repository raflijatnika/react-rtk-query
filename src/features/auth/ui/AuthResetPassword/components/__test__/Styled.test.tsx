// Test
import renderer from 'react-test-renderer'
import 'jest-styled-components'

// Components
import { StyledWrapper, StyledCard } from '../Styled'

// Constant
import {
  APP_COLOR,
  APP_COLOR_LIGHT
} from '@/features/app/constant/app-style.constant'

it('StyledWrapper should be rendered', () => {
  renderer.create(<StyledWrapper />).toJSON()
})

it('StyledWrapper should have snapshot', () => {
  const tree = renderer.create(<StyledWrapper />).toJSON()

  expect(tree).toMatchSnapshot()
})

it('StyledWrapper should have default style', () => {
  const tree = renderer.create(<StyledWrapper />).toJSON()

  expect(tree).toHaveStyleRule('height', '100%')
  expect(tree).toHaveStyleRule('display', 'flex')
  expect(tree).toHaveStyleRule('align-items', 'center')
  expect(tree).toHaveStyleRule('justify-content', 'center')
  expect(tree).toHaveStyleRule('flex-direction', 'column')
})

it('StyledCard should be rendered', () => {
  renderer.create(<StyledCard />).toJSON()
})

it('StyledCard should have snapshot', () => {
  const tree = renderer.create(<StyledCard />).toJSON()

  expect(tree).toMatchSnapshot()
})

it('StyledCard should have default style', () => {
  const tree = renderer.create(<StyledCard />).toJSON()

  expect(tree).toHaveStyleRule('background-color', APP_COLOR.WHITE)
  expect(tree).toHaveStyleRule('width', '658px')
  expect(tree).toHaveStyleRule('height', '658px')
  expect(tree).toHaveStyleRule('justify-content', 'inherit')
  expect(tree).toHaveStyleRule('flex-direction', 'inherit')
  expect(tree).toHaveStyleRule('border-radius', '10px')
  expect(tree).toHaveStyleRule('padding', '0px 80px')
})
