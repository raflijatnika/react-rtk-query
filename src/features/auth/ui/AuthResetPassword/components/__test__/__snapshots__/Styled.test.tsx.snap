// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledCard should have snapshot 1`] = `
.c0 {
  background-color: #fff;
  width: 658px;
  height: 658px;
  display: inherit;
  -webkit-box-pack: inherit;
  -webkit-justify-content: inherit;
  -ms-flex-pack: inherit;
  justify-content: inherit;
  -webkit-flex-direction: inherit;
  -ms-flex-direction: inherit;
  flex-direction: inherit;
  border-radius: 10px;
  padding: 0px 80px;
}

<div
  className="c0"
/>
`;

exports[`StyledWrapper should have snapshot 1`] = `
.c0 {
  height: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

<div
  className="c0"
/>
`;
